zerver/lib/retention.py

```diff
@@ -20,6 +20,7 @@
 log_to_file(logger, settings.RETENTION_LOG_PATH)
 
 MESSAGE_BATCH_SIZE = 1000
+TRANSACTION_DELETION_BLOCK_SIZE = 100
```

Comment:
Is this intentionally called `..._BLOCK_SIZE`, or should we be consistent and call it `..._BATCH_SIZE` like `MESSAGE_BATCH_SIZE`?